database/migrations/2021_10_10_131533_create_products_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateProductsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('sku');
+            $table->string('name');
+            $table->integer('price');
+            $table->float('tax_rate');
```

Comment:
i prefer to use decimal data type. 
as i worked before, decimal type solved many issues for financials programs as a "round".
may be use:
$table->decimal('price', 10, 2);

Reply:
also 
price, tax_rate and taxed_price

Reply:
👍